app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.errors import VaccineError
+from app.errors import NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: dict, cafe: Cafe) -> str:
+    vaccine_problem = False
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            masks_to_buy += 1
+        except VaccineError:
+            vaccine_problem = True
```

Comment:
In case `VaccineError` is raised, there is no need to reset the flag and continue execution, you could directly return the required string during the error handling.

Reply:
This comment is not resolved